catalog/spatial/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -29,12 +29,12 @@ The `HTTP GET` form of `GetCapabilities` uses query parameters via the following
 
```

Comment:
At 26 -> Should be `GetCapabilities` `HTTP GET`

![screen shot 2016-03-01 at 12 52 26 pm](https://cloud.githubusercontent.com/assets/11355332/13439833/7d6aef5a-dfac-11e5-8783-d9827e3fe49e.png)